packages/components/psammead-media-player/src/Placeholder/index.test.jsx

```diff
@@ -1,28 +1,54 @@
 import React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render, fireEvent, getByText } from '@testing-library/react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import Placeholder from '.';
 
 describe('Media Player: Placeholder', () => {
   const mockOnClick = jest.fn();
+  const mediaInfo = {
```

Comment:
Might be beneficial to add a test/snapshot which doesn't pass through `duration`, `durationSpoken` and `datetime`?